app/Livewire/GlobalSearch.php

```
@@ -0,0 +1,371 @@
+
+                    // For regular applications
+                    if ($app->fqdn) {
+                        $fqdns = collect(explode(',', $app->fqdn))->map(fn ($fqdn) => trim($fqdn));
+                    }
+
+                    // For docker compose based applications
+                    if ($app->build_pack === 'dockercompose' && $app->docker_compose_domains) {
+                        try {
+                            $composeDomains = json_decode($app->docker_compose_domains, true);
+                            if (is_array($composeDomains)) {
+                                foreach ($composeDomains as $serviceName => $domains) {
```

PHPMD (2.15.0)

Avoid unused local variables such as '$serviceName'. (undefined)

(UnusedLocalVariable)